src/jingle_sip/mod_jingle_sip.erl

```diff
@@ -360,9 +380,9 @@ make_user_header({User, _} = US) ->
 
 
 get_proxy_uri(Server) ->
-    ProxyHost = gen_mod:get_module_opt(Server, ?MODULE, proxy_host, "localhost"),
-    ProxyPort = gen_mod:get_module_opt(Server, ?MODULE, proxy_port, 5060),
-    Transport = gen_mod:get_module_opt(Server, ?MODULE, transport, "udp"),
+    ProxyHost = gen_mod:get_module_opt(Server, ?MODULE, proxy_host),
+    ProxyPort = gen_mod:get_module_opt(Server, ?MODULE, proxy_port),
+    Transport = gen_mod:get_module_opt(Server, ?MODULE, transport),
```

Comment:
Just a thought: how about getting the opts with one lookup: `gen_mod:get_loaded_module_opts(Server, ?MODULE)` and then pattern-matching the map?

Reply:
Yeah, seems like a good idea, I'll see how it looks